tensorflow/lite/kernels/conv.cc

```diff
@@ -396,6 +394,20 @@ TfLiteStatus Prepare(KernelType kernel_type, TfLiteContext* context,
     TF_LITE_ENSURE_EQ(context, NumElements(bias), SizeOfDimension(filter, 0));
   }
 
+  if (input_type == kTfLiteInt16) {
+    TF_LITE_ENSURE_EQ(context, input->params.zero_point, 0);
+    TF_LITE_ENSURE_EQ(context, output->params.zero_point, 0);
+
+    // Check we have quantized_bias_type is either kTfLiteInt64 or kTfLiteInt32.
+    if (params->quantized_bias_type != kTfLiteFloat32) {
```

Comment:
Comment from internal reviewer: "Why not kTfLiteNoType ? Can quantized_bias_type be Float32?"

Reply:
See the https://github.com/tensorflow/tensorflow/pull/58400#issuecomment-1367898150 comment for details regarding this choice. The main problem is that the FlatBuffer scheme TensorType doesn't have a NONE type which could be used as default value for older TFLite models.